sycl/test/host-interop-task/host-task-dependency.cpp

```diff
@@ -0,0 +1,199 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out %threads_lib
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Do we really need to run the executable 6 times?
What is the value of the first 3 runs?

Reply:
Removed extra runs (the first ones).